lib/stacker/stack/parameters.rb

```diff
@@ -31,6 +41,8 @@ def local
         ]
 
         available = template_defaults.merge(
+          global_vars
+        ).merge(
```

Comment:
does the order of merging matters here? if some parameter has one value in the global variable file, but another value in the region file, what value would it take? from the infra PR, looks like we are keeping `parameters` section in the region file? I'm worried about having more than once place for global parameters.

Reply:
yes, the region variables will override the global variables. it made more sense for general usecase and hierarchy, even though it's not great for enforcing on our side. I'm going to add comments to our yaml file saying which is regional vs global